src/vend-number.js

```diff
@@ -113,6 +113,34 @@ VendNumber.divide = function (...values) {
 }
 
 /**
+ * Returns the sum of all items in the collection based on property.
+ *
+ * @method sumBy
+ *
+ * @param {Collection} collection
+ *        A collection of items to loop through.
+ *
+ * @param {String} property
+ *        A property name to use for calculating the sum.
+ *
+ * @param {Number} decimalPoints
+ *        The number of decimal points to round the value to.
+ *
+ * @return {Number} the total of all items in the collection based on property.
+ */
+VendNumber.sumBy = function (collection, property, decimalPoints) {
+  let sum = 0
+  if (collection && collection instanceof Array) {
+    collection.forEach(item => {
```

Comment:
`collection.reduce` is probably technically more appropriate here but I don't see it really adding much value over what you've got so prob not worth the refactor